### ISSUE TITLE
Channel's video sorting

### ISSUE THREAD

**Issue by voiceofvraja** [TeamNewPipe/NewPipe#5202]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Ability to sort a subscription videos by basic sorting options

#### How will you/everyone benefit from this feature?

Basically just to sort them out if the user wants to sort videos by views or likes or upload date, and if the user clicks it twice it changes it by ascending or descending order

**Comment by c-shubh** [comment]
Duplicate of #2414

**Comment by mhmdanas** [comment]
@voiceofvraja can you clarify whether you want sorting for one channel's videos or for all your subscriptions?

**Comment by voiceofvraja** [comment]
@mhmdanas I meant when we open any channel whether subscribed or not, we see list of videos in grid or list. My suggestion is that we have a sort option in the 3 dots menu above through which we can choose to sort videos in ascending or descending order of views/likes/upload date. This is something which can ne applied to other places also referring to

**Comment by mhmdanas** [comment]
Then I'll be closing this in favor of #5324.
